### ISSUE TITLE
Feature Request: Add settings to adjust AWB, gain, AE etc.

### ISSUE THREAD

**Issue by dingausmwald** [irimitenkan/picam2ctrl#11]
Hey man, great work. Love it.

What do you think about exposing some more settings to home assistant? I've seen that it is possible to set some parameters with picam2.set_controls, even while udp stream is running. When not to home assistant, the config file could be a place for that.

Could be useful for people with extreme light conditions or when cameras switch from day to nightvision. Homeassistant could control those parameters based on daytime or something like that.

Sadly it does not seem to be possible to switch tuning files while running and those are rather complicated

**Comment by irimitenkan** [comment]
`What do you think about exposing some more settings to home assistant? I've seen that it is possible to set some parameters with picam2.set_controls, even while udp stream is running. When not to home assistant, the config file could be a place for that.`

Yes, would be useful and I already started with that in config file. I was very busy the last weeks to finalize and publish it, but this is definitely the next planned feature.

**Comment by dingausmwald** [comment]
Hey man. Great you added the features so quick. But i was hoping for something that could adjust the whitebalance. Colourgains and digitalgain, exposure time.

**Comment by irimitenkan** [comment]
> Hey man. Great you added the features so quick. But i was hoping for something that could adjust the whitebalance. Colourgains and digitalgain, exposure time.

Yes, I'm aware of it. #11 was only part 1 and I didn't close the ticket 😉 . In part2 I'll can add the other.
